gemrb/core/InterfaceConfig.h

```diff
@@ -61,6 +61,9 @@ struct CoreSettings {
 	path_t GemRBUnhardcodedPath;
 	path_t PluginsPath;
 	path_t GUIScriptsPath;
+#ifdef WIN32
+	bool CaseSensitive = false; // this is just the default value, so CD1/CD2 will be resolved
+#endif
 	bool CaseSensitive = true;
```

Comment:
Won't this break the windows build? Needs and #elif.

Reply:
yup, didn't notice cuz its broken in other ways also. I'll make sure all the builds work before merging.